easel/lib/screens/creator_hub/creator_hub_view_model.dart

```diff
@@ -145,19 +147,27 @@ class CreatorHubViewModel extends ChangeNotifier {
       _nftPublishedList.add(nft);
     }
 
+    if (_recentNFT != null) {
```

Comment:
What you are trying to achieve here? this method will be executed once we hit retry button. The original idea was to add it when recipe executes.